modules/v1_10_R1/src/main/java/net/countercraft/movecraft/compat/v1_10_R1/IWorldHandler.java

```diff
@@ -131,7 +130,18 @@ public void rotateCraft(@NotNull Craft craft, @NotNull MovecraftLocation originP
             setBlockFast(nativeWorld, position, Blocks.AIR.getBlockData());
         }
         //*******************************************
-        //*       Step six: Send to players       *
+        //*   Step six: Process fire spread         *
+        //*******************************************
+        for (BlockPosition position : rotatedPositions.values()) {
+            IBlockData type = nativeWorld.getType(position);
+            if (!(type.getBlock() instanceof BlockFire)) {
+                continue;
+            }
+            BlockFire fire = (BlockFire) type.getBlock();
+            fire.b(nativeWorld, position, type, new Random());
```

Comment:
Should this be a new Random instance or the random instance from the world? (for all of the occurrences)